commands/xp.py

```diff
@@ -20,15 +20,15 @@ async def xp(self, ctx, targetMember : discord.Member = None):
             if args[2].upper() == "GIVE":
                 amount = args[3]
                 if int(amount) < 0:
-                    discordError("Cannot give negative XP!", ctx)
+                    await discord_error("Cannot give negative XP!", ctx)
                 else:
                     add_exp(member.id, int(amount))
                     em = discord.Embed(description=":ok_hand: Successfully given **%s** **%s EXP**!" % (member.name, amount), color=colour.primary)
                     em.set_author(name="Operation Complete")

```

Comment:
Could this perhaps say "Success"? I'm not a big fan of "Operation Complete"

Reply:
On it